src/blocking/spi.rs

```diff
@@ -1,12 +1,30 @@
 //! Blocking SPI API
 
-/// Blocking transfer
-pub trait Transfer<W> {
+/// Blocking simultaneous read+write with separate buffers
+pub trait ReadWrite<W> {
```

Comment:
To be consistent with the async traits, can we rename `W` to `Word`?

Reply:
existing code uses a mix of `W` and `Word`. I do agree it should all be consistent, I like `W` somewhat more. It makes it more obvious that it's a generic param vs an actual type, since generic params are usually single uppercase letters.

Reply:
I'm fine with either, I just want consistency.

Reply:
My one issue with `W` is that `W` is often used for type parameters that implement `std::io::Write`.